ERROR, 2024-02-14 13:22:27, module - 'custom_logger', function - 'log', line - '66'. Message - 'Error executing SQL query: 'list' object has no attribute 'keys''
ERROR, 2024-02-14 14:07:10, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: '
ERROR, 2024-02-14 15:21:05, module - 'custom_logger', function - 'log', line - '66'. Message - 'This is an error message'
CRITICAL, 2024-02-14 15:21:05, module - 'custom_logger', function - 'log', line - '66'. Message - 'This is a critical message'
ERROR, 2024-02-14 15:21:38, module - 'custom_logger', function - 'log', line - '66'. Message - 'This is an error message'
CRITICAL, 2024-02-14 15:21:38, module - 'custom_logger', function - 'log', line - '66'. Message - 'This is a critical message'
ERROR, 2024-02-14 15:25:10, module - 'custom_logger', function - 'log', line - '66'. Message - 'This is an error message'
CRITICAL, 2024-02-14 15:25:10, module - 'custom_logger', function - 'log', line - '66'. Message - 'This is a critical message'
ERROR, 2024-02-14 15:26:52, module - 'custom_logger', function - 'log', line - '66'. Message - 'This is an error message'
CRITICAL, 2024-02-14 15:26:52, module - 'custom_logger', function - 'log', line - '66'. Message - 'This is a critical message'
ERROR, 2024-02-14 15:33:50, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'id''
ERROR, 2024-02-14 15:33:50, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'id''
ERROR, 2024-02-14 15:34:05, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'id''
ERROR, 2024-02-14 15:34:05, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'id''
ERROR, 2024-02-14 15:34:34, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'id''
ERROR, 2024-02-14 15:34:34, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'id''
ERROR, 2024-02-14 15:34:47, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'rate''
ERROR, 2024-02-14 15:34:47, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'student_id''
ERROR, 2024-02-14 15:35:10, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'rate''
ERROR, 2024-02-14 15:35:10, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'student_id''
ERROR, 2024-02-14 15:35:42, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'rate''
ERROR, 2024-02-14 15:35:42, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'student_id''
ERROR, 2024-02-14 15:35:53, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'rate''
ERROR, 2024-02-14 15:35:53, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: 'student_id''
ERROR, 2024-02-14 16:49:17, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: list index out of range'
ERROR, 2024-02-14 16:49:41, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: list index out of range'
ERROR, 2024-02-14 16:50:28, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: list index out of range'
ERROR, 2024-02-14 16:52:13, module - 'custom_logger', function - 'log', line - '66'. Message - 'while executing query: list index out of range'
